client/src/js/addendum/AddendumForm.jsx

```diff
@@ -0,0 +1,284 @@
+          return
+        }
+        setError('Request failed, please try again later')
+      })
+      .finally(() => {
+        markUpdateStatus(false)
+        setAddendums(addendums => [...addendums, addendum])
+      })
+  }
+
+  const setAddedIdentity = (identity) => {
+    setAddedIdentities(addedIdentities => [identity, ...addedIdentities])
+    markUpdateStatus(true)
+  }
+
+  const setRemovedIdentity = (identity) => {
+    return (evt) => {
+      evt.preventDefault()
+      setRemovedIdentities(removedIdentities => [identity, ...removedIdentities])
+      _.remove(activeIdentities, identity)
+      setActiveIdentities(activeIdentities)
+      markUpdateStatus(true)
+    }
+  }
+
+  const undoRemove = (identity) => {
+    return (evt) => {
+      evt.preventDefault()
+      _.remove(removedIdentities, identity)
+      setRemovedIdentities(removedIdentities)
+      setActiveIdentities(activeIdentities => [identity, ...activeIdentities])
+    }
+  }
+
+  const undoAdd = (identity) => {
+    return (evt) => {
+      evt.preventDefault()
+      setAddedIdentities(_.without(addedIdentities, identity))
+    }
+  }
+
+  const markUpdateStatus = (value) => {
+    props.updateStatus(value)
+  }
+
+  const headers = {}
+  headers[AddendumType.CONTRIBUTOR] = 'Active Contributors for this Agreement'
+  headers[AddendumType.COSIGNER] = 'Active Managers for this Agreement'
+
+  const formTitle = {}
+  formTitle[AddendumType.CONTRIBUTOR] = 'Contributors'
+  formTitle[AddendumType.COSIGNER] = 'Managers'
+
+  const identityType = {}
+  identityType[AddendumType.CONTRIBUTOR] = 'Contributor'
+  identityType[AddendumType.COSIGNER] = 'Manager'
+
+  const subtitle = {}
+  subtitle[AddendumType.CONTRIBUTOR] = 'Here is a list of Users/Identities that are authorized to make contributions to ONF projects under this agreement'
+  subtitle[AddendumType.COSIGNER] = 'Here is a list of Users/Identities that are allowed to add and remove Contributor Identities to/from this Agreement'
+
+  const text = {}
+  text[AddendumType.CONTRIBUTOR] = 'Users/Identities listed under "Contributors" for this Agreement will be allowed to make contributions to ONF projects.'
+  text[AddendumType.COSIGNER] = 'Users/Identities listed under "Managers" for this Agreement will be allowed to add and remove Contributor ' +
+    'Identities to/from this Agreement AND to add and remove other Managers to/from this Agreement.'
+
+  const updateForm = (
+    <div className={classes.updateFormContainer + ' AddendumContainer__update-form '}>
+      <Grid item xs={12}>
+        <Grid container spacing={2}>
+
+          <Grid item xs={12}>
+            <h2>Update {formTitle[props.addendumType]}</h2>
+            <Box>
+              <p className={classes.bold}>
+                {text[props.addendumType]}
+              </p>
+              <p>
+                Use the form below to add {identityType[props.addendumType]} Identities, or select one from the above list to remove it.
```

Comment:
nit
```suggestion
                Use the form below to add {identityType[props.addendumType]} identities, or select one from the above list to remove it.
```

Reply:
@aindermitte provided the text and was explicit about maintaining the capitalization

Reply:
Ok, I trust @aindermitte capitalization choices ;)